crates/matrix-sdk-crypto/src/olm/group_sessions/outbound.rs

```diff
@@ -69,7 +69,7 @@ const ROTATION_MESSAGES: u64 = 100;
 pub(crate) enum ShareState {
     NotShared,
     SharedButChangedSenderKey,
-    Shared(u32),
+    Shared(u32, u32),
```

Comment:
This could take some docs, a single u32 could be figured out relatively easily, two of them double the confusion. It probably makes sense to convert the enum variant from a tuple variant to one with named fields, i.e.:

```rust
Shared {
    megolm_index: u32,
    wedging_foo: u32,
}
```